nbconvert/preprocessors/tests/files/Parallel Execute.ipynb

```diff
@@ -0,0 +1,84 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import os.path\n",
+    "import tempfile\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "other_notebook = {'A':'B', 'B':'A'}[this_notebook]\n",
```

Comment:
Might be good to add a comment about how this works with the A waiting for B file and visa versa

Reply:
Also you probably want a comment that this_notebook is undefined and injected by the test

Reply:
Thanks